CHANGELOG.md

```diff
@@ -16,17 +16,27 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 - The operator now parses links from the compliance content and renders it in
   custom resources accordingly.
 
+- Optimize how we check the KubeletConfig rule, we now store the runtime KubeletConfig
+  in a ConfigMap per node when a node scan is launched. Then, we mount the ConfigMap to
+  the scanner pod to scan for it. Hold on to applying remediation until all scans are 
+  done in the suite.
+  This fixes issues when comparing the KubeletConfig for each node.
+  This also fixes "/api/v1/nodes/NODE_NAME/proxy/configz" warning message in the log.
+  [OCPBUGS-11037](https://issues.redhat.com/browse/OCPBUGS-11037)
```

Comment:
We will need to consider providing a migration here so that people who were using TailoredProfiles that excluded a specific platform rule don't experience an issue when they upgrade because the rule is now a node rule again.

We'll need to make sure we document it, but also provide some tooling since we're v1.0.0 now.

Reply:
Not sure about OLM (except for infra to provide webhooks but I don't think we should go the webhook way unless necessary). But maybe we could add bespoke code to the SSB controller that would just rewrite the SSB and throw a deprecation warning and remove the warning after a release or two?

Reply:
that's a good advice! I think I can do that

Reply:
Since we are not going to have a composite profile, let's create a doc to address this issue